templates/security/cve/_cve-table.html

```diff
@@ -1,83 +1,52 @@
-<table class="cve-table">
+<table aria-label="Detailed view of CVE card" id="table--{{ cve.id }}">
   <thead>
     <tr>
-      <th style="width: 10em;">ID</th>
-      <th style="width: 8em;">Priority</th>
-      <th style="width: 8em;">Package</th>
-        {% for release in selected_releases %}
-          <th style="width: 8em;">
-            <div class="icon-container__text">
-              {{ release.version }} {{ release.support_tag }}
-            </div>
-          </th>
-        {% endfor %}
+      <th>Package</th>
+      {% if versions|length == 0 %}
+        <th>24.04 LTS</th>
+        <th>23.10 LTS</th>
+        <th>22.04 LTS</th>
+        <th>20.04 LTS</th>
+        <th>18.04 ESM</th>
+      {% else %}
+        {% for release in selected_releases|reverse %}<th>{{ release.version }} {{ release.support_tag }}</th>{% endfor %}
+      {% endif %}
     </tr>
   </thead>
   <tbody>
-    {% for cve in cves %}
-      {% for package in cve.packages %}
-        <tr>
-          {% if loop.index == 1 %}
-            <td class="cve-table-cell-id" rowspan="{{ cve.packages | length }}">
-              <a href="/security/{{ cve.id }}">{{ cve.id }}</a>
-            </td>
-            <td class="cve-table-cell-priority" rowspan="{{ cve.packages | length }}">
-              <div class="icon-container__icon">
-                {% if cve.priority == 'unknown' %}
-                  <i class="p-icon--placeholder"></i>
-                {% elif cve.priority == 'negligible' %}
-                  <i class="p-icon--negligible-priority"></i>
-                {% elif cve.priority == 'low' %}
-                  <i class="p-icon--low-priority"></i>
-                {% elif cve.priority == 'medium' %}
-                  <i class="p-icon--medium-priority"></i>
-                {% elif cve.priority == 'high' %}
-                  <i class="p-icon--high-priority"></i>
-                {% elif cve.priority == 'critical' %}
-                  <i class="p-icon--critical-priority"></i>
-                {% else %}
-                  <i class="p-icon--placeholder"></i>
-                {% endif %}
-              </div>
-              <div class="icon-container__text">
-                <span class="capitalize">{{ cve.priority }}</span>
-              </div>
-              <div class="icon-container__icon icon-cve-info">
-                <i class="p-icon--information cve-tooltip-icon cve-tooltip-icon-priority" data-priority="{{ cve.priority }}"></i>
-              </div>
+    {% for package in cve.packages %}
+      <tr class={% if loop.index > 5 %}"expandable-row u-hide"{% endif %}>
+        <td>{{ package.name }}</td>
+        {% for release in selected_releases|reverse %}
```

Comment:
I believe `selected_releases` is only used on the table, we can probably refactor the way these are selected in the view so that you get the correct list without having to do these checks for the codename here